### DISCUSSION TITLE
Jumpt to line number?

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [helix-editor/helix#1251] [Q&A] [answered]
How can I jump to a specific line number?

**Comment by NNBnh** [comment, marked as answer]
`<n>` <kbd>g</kbd> <kbd>g</kbd> => Goto line `<n>` <sup>th</sup>.

**Reply by kirawi** [reply]
Alternatively `<n>` `G`

**Reply by Robert-M-Muench** [reply]
> g | Go to the start of the file

I think the docs are miss leading. How am I supposed to understand that I can hack in a number before pressing `gg`?

I think `G` is missing from the key-map page.

**Reply by pickfire** [reply]
> I think the docs are miss leading. How am I supposed to understand that I can hack in a number before pressing gg?

True, any idea how you think it can be better? My idea is `Go to the start of the file or line number if count is given` or `Go to the start of the file or <n> line number`, not sure if it can be shorter since it is long and will make the infobox ugly.

**Reply by Robert-M-Muench** [reply]
`Go to line number <n> else start of file`

**Reply by eugenesvk** [reply]
is there a way to enter `<n>` after running the command? I find it a bit more intuitive vs. having to set the line number in advance

**Reply by Omnikar** [reply]
For `gg` and `G`, no, but there's another way to jump to a specific line that wasn't mentioned here that does have you type the number after: type the line number like a command in command mode; `:<n>`.

**Comment by oiwn** [comment]
Yea it's not clear from docs, when i press g and next number like 100 nothing happened. I really do not know how to make g command work.

**Comment by David-Else** [comment]
`g` `<n>` `g` also works, just to make it a bit more confusing :)

**Comment by 7flash** [comment]
@oiwn @David-Else it's not confusing at all, but very much in-line with helix philosophy, first you choose number then you choose action.

**Comment by plutack** [comment]
I have to agree with @Robert-M-Muench and @oiwn. Why not include @NNBnh suggestion to the docs? If I have to come to a github discussion after reading about something in the docs, it is one of three things; I missed something in the docs prior to the line that "doesn't work", I am dumb😅 or it isn't clear enough. I hope it is not the 2nd one.

**Comment by adriangalilea** [comment]
Very new to this, never even used vim.

How do I jump to a relative line number?

**Reply by plutack** [reply]
The relative number then j for down or k for up. I strongly advise you go through the tutor

**Reply by adriangalilea** [reply]
I literally just went through the tutor, some times were hard to grasp for a mouse cave-man user like me :)

Thanks!

**Comment by hippie68** [comment]
I suggest for the maintainers to implement `<n>g`. For me the alternatives `<n>gg`, `g<n>g`, and `<n>G` appear unnecessarily cumbersome. I think implementing `<n>g` should be possible, because it seems the only way to use a number with Goto is the simple `Goto line number <n>`.

This addition would be in line with the existing `<n>j` and `<n>k`.

**Comment by plutack** [comment]
I have reconciled with gg...because I think it is on line with bindings like gl gh
